Misc/NEWS.d/next/Library/2019-05-10-19-27-07.bpo-28724.rHcm24.rst

```diff
@@ -0,0 +1 @@
+The socket module now has :func:`send_fds()` and :func:`recc_fds()` methods.
```

Comment:
Add "Patch by Shinya Okano and Shinya Okano" at the end.

Reply:
Noted. Thanks